### ISSUE TITLE
[Bug] Widget does not refresh automatically 

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#14206]
## Steps to reproduce
1. Install the search widget with Google app enabled on your device
2. Disable the Google app right before the widget update, hourly job run. 
3. Inspect the "voice icon/mic icon" on the search widget.
### Expected behavior
Mic icon is removed from widget, when updated, after the run of the hourly job service.
### Actual behavior
Mic icon is not automatically removed after being updated, after the hourly job service.
### Device information

* Android device: Google Pixel 3 XL (Android 9), Samsung Galaxy S9 (Android 8)
* Fenix version: Nightly 8/26

Note:
1. As an example, in my case, i disabled the google app with a couple of minutes before having the job run, which should be around 10:00, in this situation. It can be seen that the mic icon is still present, for the widget that was set before the job run.
I've also added a new widget after having google app disabled for which it can be seen that the mic icon is not displayed.(please check screen)

<details><summary>Screen</summary>

![Screenshot_20200826-102747_Samsung Experience Home](https://user-images.githubusercontent.com/49859863/91274180-faf1b580-e786-11ea-8bc6-7166c1bf2f87.jpg)
</details>

**Comment by Mugurell** [comment]
Hey Diana,
Just curious, why did you expect the refresh to happen at 10:00?
We register to the framework to be refreshed every hour, but there is no fix hour at which the refresh should happen.
A better test should be:
- add the widget
- disable google
- check back after a bit more than an hour (Android does not guarantee complete accuracy).

**Comment by Diana-Rus** [comment]
Hi, rechecked with the provided info on Nightly 9/7 with:
- Google Pixel 3XL (Android 9)
- Samsung Galaxy S9 (Android 8)

I've checked back after a bit more than an hour, maybe with 10 minutes or so (after having the widget added and google disabled) and the widget has been updated.


> Just curious, why did you expect the refresh to happen at 10:00?

Yes, didn't know exactly what the trigger was for the widget hourly job.

I'll be closing this based on the information from https://github.com/mozilla-mobile/fenix/issues/14206#issuecomment-681582488 as it seems it works correctly.
